lib/order.rb

```diff
@@ -0,0 +1,45 @@
+require_relative 'menu'
+require_relative 'send_sms'
+require 'twilio-ruby'
+
+class Order
+  NOT_ON_MENU_ERROR = 'This is not on the menu'.freeze
```

Comment:
I always forget to freeze mutable objects in constants! Need to follow your lead